polylogue/cli/init.py

```
@@ -0,0 +1,154 @@
+        drive_setup = ui.confirm("  Set up Drive credentials?", default=False)
+    else:
+        console.print("  Skipping in plain mode (run 'polylogue sync drive' later to set up)")
+
+    if drive_setup:
+        try:
+            drive_client = DriveClient(ui)
+            drive_client.ensure_credentials()
+            console.print("[green]✓ Drive credentials configured")
+        except (SystemExit, KeyboardInterrupt):
+            console.print("[yellow]Drive setup skipped")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)